grantnav/frontend/templates/components/grants_table_scripts.html

```diff
@@ -79,8 +77,25 @@
           {data: "awardDate", orderable: false},
           {data: "fundingOrganization.0.name", render: function (data, type, row) {return '<a href="/org/' + row.fundingOrganization[0].id + '">' + truncate(data, 30) + '</a>'}, orderable: false},
           {data: "recipientOrganization.0.name", render: function (data, type, row) {return '<a href="/org/' + row.recipientOrganization[0].id + '">' + truncate(data?data:row.recipientOrganization[0].id, 30) + '</a>'}, orderable: false},
-          {data: "description", orderable: false},
+          {data: "description", orderable: false, responsivePriority: -1},
         ]
-      });
+    });
+
+    const table = $('#search_grants_datatable').DataTable()
+    table.on('responsive-display', function ( e, datatable, row, showHide, update ) {
+    });
```

Comment:
Think this might be a debug function left in? Doesn't do any harm but have raised https://github.com/ThreeSixtyGiving/grantnav/issues/937 as a reminder to remove it if it is not needed.